### ISSUE TITLE
Don't display fraction of numbers if they are equals to zero

### ISSUE THREAD

**Issue by Insality** [defold/defold#9412]
**Is your feature request related to a problem? Please describe (REQUIRED):**
Before editor update (Defold 180), in the Editor property number values displayed without fraction, if they equals to zero. So the Number `0` displayed as `0`. In new Editor (192 for example) this value will be displayed as `0.0`.

**Describe the solution you'd like (REQUIRED):**
Don't display `.0` at number properties

**Additional context (OPTIONAL):**
Here two screenshots to compare
![Screenshot 2024-09-10 at 22 03 30](https://github.com/user-attachments/assets/776ef401-9478-4377-b1db-688920720287)
![Screenshot 2024-09-10 at 22 02 09](https://github.com/user-attachments/assets/9f3b89c2-4829-4f79-875d-48852b685a9b)

**Comment by Insality** [comment]
But from user perspective is harder to read

**Comment by AGulev** [comment]
But it's easier to understand what values expected in what property fields.

**Comment by Insality** [comment]
I don't agree with this statement. On my mind it should be more informative and easy to read at glance.

I spend a lot of time at GUI scene editor and just share my thoughts that this change make my life a little bit harder than before

**Comment by Insality** [comment]
Coping my message from discord chat:

I think it's a place that should be more readable, since we spend a lot of time here. Users will know with time by manipulation objects with gizmo (transform/scale gizmos) that they can be a "float" (ex. 1.1 for scale property)

it's not a right place to be more technical on my point of view. We can check any other softwares like Figma, it also don't show .0 postfix

**Comment by AGulev** [comment]
> check any other softwares

not any, it depends:
blender
![CleanShot 2024-09-11 at 18 16 11@2x](https://github.com/user-attachments/assets/396175b5-6ea2-4555-b108-6bfe619f864a)
spine
![CleanShot 2024-09-11 at 18 15 26@2x](https://github.com/user-attachments/assets/9c21f17c-8f5b-4a96-a619-075546b23767)
tiled
![CleanShot 2024-09-11 at 18 21 22@2x](https://github.com/user-attachments/assets/43e9e43c-ac9c-4a67-94fa-8376d082f53b)
DaVinci Resolve
![CleanShot 2024-09-11 at 18 25 30@2x](https://github.com/user-attachments/assets/75bf48fb-e29d-4d42-83a2-2ada22a9e0d7)
Inscape
![CleanShot 2024-09-11 at 18 27 37@2x](https://github.com/user-attachments/assets/1c58b2f6-ccba-4d27-b6bb-0b65e254dacb)

I think I don't anything else on my mac, so I asked Tati to take a look her tool on ipad:
Afinity designer
![CleanShot 2024-09-11 at 18 38 09@2x](https://github.com/user-attachments/assets/66ed5926-fb71-477e-88f6-284e6e9e15a1)

ProCreate
![CleanShot 2024-09-11 at 18 39 15@2x](https://github.com/user-attachments/assets/cafabb11-809b-4d00-8ff2-59b1a46a5200)

**Comment by Insality** [comment]
In a bunch of these software I think it will be much better to have integers in a place where it possible (spine, davinci)

I'm here to share that is an issue for me and it better to work with a lot of properties especially in scene/GUI layout without a lot of dots and zeros

**Comment by AGulev** [comment]
> In a bunch of these software I think it will be much better to have integers in a place where it possible (spine, davinci)
> 
> I'm here to share that is an issue for me and it better to work with a lot of properties especially in scene/GUI layout without a lot of dots and zeros

Yes, sure. That's why we have this issue tracker. 
But I'm augmenting my position why I disagree with you.

All the options should be taken into consideration to make a decision. 
I'm not pushing my opinion as the only right, I'm just explaining why I think it is right.

**Comment by aglitchman** [comment]
My two cents.

If the number is floating point, it should be explicitly visible. It's fine now.

All that can be improved is to gray out the fractional part if it is zero. The readability will improve a bit.

**Comment by matgis** [comment]
Hiding the fractional part could maybe be added as a user preference in the future, but I think seeing a field accepts float values is quite important for discoverability.

**Comment by britzl** [comment]
We are showing fractional numbers by design. This will not be changed.
